modules/admin_manual/pages/release_notes.adoc

```diff
@@ -22,103 +22,145 @@
 
 == Changes in 10.2.0
 
-Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.2 that need your attention. You can also read https://owncloud.org/changelog/server/[the full ownCloud Server changelog] for further details on what has changed.
+Dear ownCloud administrator, please find, below, the changes and known issues in ownCloud Server 10.2 that need your attention. 
+You can also read https://owncloud.org/changelog/server/[the full ownCloud Server changelog] for further details on what has changed.
 
 === Migrations
 
 Please note that this minor release contains database migrations which impact the upgrade duration.
 Specifically:
 
-* The `oc_share` table receives a new column (impact depends on the number of shares)
-* The `oc_authtoken` table's login name column size has been increased (impact depends on the number of recently logged in users plus several app passwords)
+* The `oc_share` table has a new column. The impact of this change depends on the number of shares in your ownCloud installation.
```

Comment:
The impact is what?
runtime degredation?
migration time?
...?

Reply:
Good question. @pmaier1?

Reply:
As in the sentence above "this minor release contains database migrations which impact the upgrade duration". Do we need to make this more clear?

Reply:
I have reworded the sentence to make it more clear.